### PR TITLE
Remove Opaque and Depth Textures from Overlay Camera properties

### PR DESCRIPTION
**Pull request by berniegp**
### Purpose of this PR
The documentation says that the Opaque Texture and Depth Texture properties are used for Overlay cameras. This is not true AFAICT and the opposite is said in `Graphics/com.unity.render-pipelines.universal/Documentation~/camera-component-reference.md`.

---
### Testing status
n/a

---
### Comments to reviewers
I do not know if this should be backported and I don't think this needs a changelog entry. Please note that I'm external to the Graphics team.

### CODE REVIEW

**Comment by berniegp** [comment]
I don't know how to cancel Yamato jobs and the link given in the PR template (http://go/graphics-yamato-faq) does not work for me.

**Review by phi-lira — APPROVED** [review]

**Comment by berniegp** [comment]
Silly me, I wasn't on VPN. Now the link works!
@phi-lira : should I wait on @oleks-k to review this since it's a doc change or just go ahead and merge? Should I backport this to other releases?

**Review by simon-engelbrecht-soerensen — APPROVED** [review]

**Comment by phi-lira** [comment]
Should be backported. I poked @oleks-k otherwise good to go.
@berniegp After changes are approved you can merge yourself.

**Review by oleks-k — APPROVED** [review]
Looks good, thanks for spotting this!

**Comment by berniegp** [comment]
Thanks. Am I expected to do the backport or are you on it?

**Pull request merged**
